frontend/src/exams/add-exam-form-modal.vue

```diff
@@ -265,6 +266,7 @@
         this.resetCaptureTab()
         this.unSubmitted = true
         this.submitMsg = ''
+        this.status = 'unknown'
```

Comment:
What is the unknown status?

Reply:
unknown is the default state status before the form is submitted.  It triggers the spinner loader thing to display after submission and gets modified by the promise to postExam.  It'll either get set to "success" or "failed" depending on the outcome of the HTTP request.  Maybe "unknown" was an odd choice of word but I figured it is unknown until the outcome is determined.